### DISCUSSION TITLE
Game/app detection with a status, and an overlay

### DISCUSSION THREAD

**Discussion by Steveplays28** [stoatchat/discussions#233] [Feature Suggestions]
Would be cool, shhh totally didn't steal this from somewhere :)

**Comment by Sventra** [comment]
I don't like any kind of overlay, they usually are resource hungry bloat that more often than not cause issues with performance/anticheats, I immediately disable it every time I have to reinstall any piece of software that comes with one

**Reply by exdnaa** [reply]
I can't share your opinion.  Many people like this feature others don't.  That is normal.  If the function really appears, you can definitely turn it on and off.

**Reply by Steveplays28** [reply]
It can be an opt-in feature :p

**Reply by erkinalp** [reply]
Game detection requires a compatible client, that Revolt cannot force its users to use.

**Reply by Rexogamer** [reply]
It would not be forced on users.

**Comment by aurkaxi** [comment]
+1 for overlay. it really helps. yes it is resource hungry, so if you don't want just disable it. there are tons of people who don't want to switch application for a short operation.
